src/app/containers/MostRead/label.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { useContext } from 'react';
```

Comment:
PR looks good to me :D Could we just add unit tests for this component, that ensures we render a label with the correct id and header?

Reply:
tests added